designer/server/src/test/resources/logback-test.xml

```diff
@@ -7,10 +7,12 @@
         </encoder>
     </appender>
 
-    <root level="WARN">
+    <root level="INFO">
```

Comment:
At some point in time, we had a problem that we were hitting limit of logs in GH actions. I wonder if this problem won't back after this change